test/tree/ast/typescript/typeScriptFileParserProject.test.ts

```diff
@@ -47,7 +47,7 @@ describe("TypeScriptFileParser real project parsing: TypeScript", () => {
                 assert(values.length > 2);
                 done();
             }).catch(done);
-    }).timeout(5000);
```

Comment:
These changes don't relate to this PR, but were necessary due to slow internet while working on this.